examples/python/Quantinuum_conditional_execution_h_series.py

```diff
@@ -0,0 +1,73 @@
+# <div style="text-align: center;">
+# <img src="https://assets-global.website-files.com/62b9d45fb3f64842a96c9686/62d84db4aeb2f6552f3a2f78_Quantinuum%20Logo__horizontal%20blue.svg" width="200" height="200" /></div>
+
+# # Conditional Execution on H-Series
+
+# Quantinuum H-Series systems are capable of executing conditional workflows. This notebook presents how to do this in pytket. For more information, see [Conditional Execution](https://tket.quantinuum.com/examples/conditional_gate_example.html?highlight=qasm) and [Classical and conditional operations](https://cqcl.github.io/pytket/manual/manual_circuit.html#classical-and-conditional-operations).
```

Comment:
The manual link is now -> https://tket.quantinuum.com/user-manual/manual_circuit.html#classical-and-conditional-operations